server/policies/collection.test.ts

```diff
@@ -40,7 +40,11 @@ describe("admin", () => {
       teamId: team.id,
       permission: CollectionPermission.Read,
     });
-    const abilities = serialize(user, collection);
+    // reload to get membership
+    const reloaded = await Collection.scope({
+      method: ["withMembership", user.id],
+    }).findByPk(collection.id);
```

Comment:
Reload wasn't required earlier because of early return within policy check, preventing control to invoke `includeMemberships(...)`. Now, with `and...or` pattern it becomes mandatory.